testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4WD/sandy/config_driver.xml

```diff
@@ -0,0 +1,8 @@
+<driver_script name="run_test.py">
+	<case name="init">
+		<step executable="./run.py" quiet="true" pre_message=" * Running init" post_message="     Complete"/>
+	</case>
+	<case name="forward_RK4">
+		<step executable="./run.py" quiet="true" pre_message=" * Running forward RK4" post_message="     Complete"/>
+	</case>
```

Comment:
If you agree that the RK4 descriptor isn't needed, this would need to be changed also.